src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.DisableStandardFrameworkResolution.targets

```diff
@@ -15,6 +15,14 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="GetFrameworkPaths" />
 
   <PropertyGroup>
+    <TargetFrameworkMonikerDisplayName Condition="'$(TargetFramework)' == 'netstandard2.0'">.NET Standard 2.0</TargetFrameworkMonikerDisplayName>
+    <TargetFrameworkMonikerDisplayName Condition="'$(TargetFramework)' == 'netstandard2.1'">.NET Standard 2.1</TargetFrameworkMonikerDisplayName>
+    <TargetFrameworkMonikerDisplayName Condition="'$(TargetFramework)' == 'netcoreapp3.1'">.NET Core 3.1</TargetFrameworkMonikerDisplayName>
+    <TargetFrameworkMonikerDisplayName Condition="'$(TargetFramework)' == 'net5.0' OR '$(TargetFramework)' == 'netcoreapp5.0'">.NET 5.0</TargetFrameworkMonikerDisplayName>
+    <TargetFrameworkMonikerDisplayName Condition="'$(TargetFramework)' == 'net6.0' OR '$(TargetFramework)' == 'netcoreapp6.0'">.NET 6.0</TargetFrameworkMonikerDisplayName>
+    <TargetFrameworkMonikerDisplayName Condition="'$(TargetFramework)' == 'net7.0' OR '$(TargetFramework)' == 'netcoreapp7.0'">.NET 7.0</TargetFrameworkMonikerDisplayName>
```

Comment:
This logic should be in terms of the `TargetFrameworkIdentifier` and `TargetFrameworkVersion` properties, and should use MSBuild built-in version comparison functions such as `VersionGreaterThanOrEquals` as appropriate so that this doesn't need to be updated for each new version of .NET.  Here is an example of similar logic for implicit compilation symbols: https://github.com/dotnet/sdk/blob/d8fac8888aca6f7365019a5d97f9658b39f9713e/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets#L177-L191

For example, you could hard code the .NET Standard and .NET Core versions, and then for .NET 5 and above use something like the following:

```xml
<TargetFrameworkMonikerDisplayName Condition="''$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), 5.0))">.NET $(TargetFrameworkVersion)</TargetFrameworkMonikerDisplayName>
```

Reply:
Wait so there is a way to test this? Unless you are referring to a way to invoke msbuild in the unit test and checking the output ``AssemblyAttributes.cs`` file that is output from a task?

Reply:
After thinking about this, perhaps I should have it check .NET Core 1.0 all the way to 3.1 in the same line, then on .NET Standard 1.0 to 2.1 on a single line as well.